Kernel/Syscalls/unveil.cpp

```diff
@@ -90,6 +83,25 @@ int Process::sys$unveil(Userspace<const Syscall::SC_unveil_params*> user_params)
         }
     }
 
+    // Now, let's try and resolve the path and obtain custody of the inode on the disk, and if not, bail out with
+    // the error from resolve_path_without_veil()
+    // However, if the user specified unveil() with "c" permissions, we don't set errno if ENOENT is encountered,
+    // because they most likely intend the program to create the file for them later on.
+    // If this case is encountered, the parent node of the path is returned and the custody of that inode is used instead.
+    RefPtr<Custody> custody_on_error; // Parent inode in case of ENOENT
```

Comment:
A `custody_or_error` should be of type `KResult<NonnullRefPtr<Custody>>`. Either rename the variable or change the type :)

Reply:
Ah, it's custody *on* error!

That's very hard to see given how the `_or_error` names are used everywhere else.

Why not just name it `parent_custody` or something?

Reply:
> Why not just name it parent_custody or something?

Good idea, I was having trouble coming up with a name that would describe what this variable actually related to, but that should make the most sense, considering someone can just look at the definition of `resolve_path_without_veil()`